flipperdevices/flipperzero-firmware
# [FL-3847, FL-3513] Thread Signals

# What's new

- Add a signal API for threads
- Integrate signals into the `Loader` and `EventLoop` subsystems
- Add `loader close` CLI subcommand
- Show the application name when `loader info` is executed (FL-3513)

# Verification 

1. Start any `EventLoop`-based application, for example: `loader open NFC`
2. Run the `loader info` command. The application name should be printed.
3. Run the `loader close` command. The application should exit and the corresponding message should be printed.
4. Start any application that does not use the `EventLoop` subsystem.
5. Repeat step 3. This time, an error message should be printed.

# Checklist (For Reviewer)
